js/test/test.js

```diff
@@ -419,11 +410,8 @@ async function testExchange (exchange) {
 async function runPrivateTests(exchange, symbol) {
     exchange.checkRequiredCredentials ();
 
-    await testMethod ('signIn', exchange);
```

Comment:
why this was removed?

Reply:
@ttodua ok I will restore it I was assuming we were going to run them sequentially and added it inside `tests`